### ISSUE TITLE
Composer error

### ISSUE THREAD

**Issue by carlessebastian** [Recombee/php-api-client#3]
When executing composer require recombee/php-api-client it return error requiring rmccue/requests ^1.7.1 no matching package found (last version 1.7.0).

Any hint?
Thanks in advance,
Carles

```
root@dev-16-inp-web:/data/recombee-test/php-api-client (master)$ composer require recombee/php-api-client
Using version ^1.3 for recombee/php-api-client
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Can only install one of: recombee/php-api-client[v1.3, dev-master].
    - Can only install one of: recombee/php-api-client[v1.3.2, dev-master].
    - recombee/php-api-client v1.3.1 requires rmccue/requests ^1.7.1 -> no matching package found.
    - Installation request for recombee/php-api-client dev-master -> satisfiable by recombee/php-api-client[dev-master].
    - Installation request for recombee/php-api-client ^1.3 -> satisfiable by recombee/php-api-client[v1.3, v1.3.1, v1.3.2].

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://getcomposer.org/doc/04-schema.md#minimum-stability> for more details.

Read <https://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.
```

**Comment by OndraFiedler** [comment]
This bug should be fixed in version 1.3.2, could you please try explicitly this version?

**Comment by OndraFiedler** [comment]
Did using of the lastest version help?
